ecl/eclscheduler/eclscheduler.cpp

```diff
@@ -172,14 +154,185 @@ void openLogFile()
 
 //=========================================================================================
 
-static constexpr const char * defaultYaml = R"!!(
+static constexpr const char *defaultYaml = R"!!(
 version: "1.0"
 eclscheduler:
   daliServers: dali
   enableSysLog: true
   name: myeclscheduler
 )!!";
 
+//=========================================================================================
+
+class EclSchedulerServer : implements CSimpleInterfaceOf<IAbortHandler>
```

Comment:
trivial/style: should be "public" not "implements" really, just because CSimple.. is not a pure abstract class (i.e. not an interface) and we reserve implements for interfaces only.